exercicios/para-casa/.env.example

```
@@ -0,0 +1,2 @@
+PORT=
+MONGO_DB=
```

só temos essas duas variáveis de ambiente? talvez pudessemos adicionar o SECRET, oq acha?
```suggestion
MONGO_DB=
SECRET=
```

---

@mflilian, May eu agradeço sua revisão. Já atualizei as variáveis de ambiente adicionando o SECRET ^^